Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -172,8 +148,9 @@ class PreplannedMapModel: ObservableObject, Identifiable {
                 parameters: parameters,
                 downloadDirectory: mmpkDirectory
             )
-            JobManager.shared.jobs.append(job)
-            startAndObserveJob(job)
+            
+            OfflineManager.shared.start(job: job)
```

Comment:
The models that kick off jobs now need to do that through the OfflineManager and not use the JobManager at all.